src/core/liberation_bold_italic_widths.js

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2021 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Widths of glyphes in LiberationSans-BoldItalic.ttf.
+const LiberationBoldItalicWidths = [
```

Comment:
Would it not be better to call this `LiberationSansBoldItalicWidths`, and similarly for the other fonts below, to reduce the risk that we'd need to rename everything in the future if e.g. support for *serif* Liberation fonts is added?

Reply:
Also, given that these arrays aren't *that* large, how about simply placing all of them in just one file instead (e.g. `liberation_sans_widths.js`)?